week-4/A-objects-intro/README.md

```diff
@@ -1,8 +1,23 @@
-### Goals
-
-- introduce objects as a concept
-- when can we use objects (to store what type of information)
-- the very specific object syntax (the commas after each property, the colon between property name and value, the curly brackets)
-- good practices for naming giving names to properties (not having spaces, what to do when there is a space)
-- the language to use when talking about an objects: object literal, properties, property names, property values
-- very simple exercises to get them used to declaring object literals
+### Objects
+
+When you think about real world objects, they have a bunch of properties that describe how its unique. Your laptop, for example, has a brand (Lenovo/Apple etc.), a screen size (13/15 inch), RAM (8/16GB) etc.

```

Comment:
One of the aspirations of the repo is to write the lessons in [Simple English](https://simple.wikipedia.org/wiki/Wikipedia:Simple_English_Wikipedia#Simple_English) (short sentences, limited vocab, simple grammar etc.)

The lesson is well written and understandable but we can make some of the sentences simpler. For example:

> Objects in the real world have properties that describe how they are unique.

Reply:
Good comments! Please keep them coming if you see anything else, especially the "language/use of english" ones!